pkg/autoops/storage/v2/sql/auto_ops_rule/update_auto_ops_rule.sql

```diff
@@ -4,7 +4,7 @@ SET
     feature_id = ?,
     ops_type = ?,
     clauses = ?,
-    triggered_at = ?,
+    triggered_at = 0,
```

Comment:
There is no need to set this to zero.
You can remove this line from the query because the field is not required when updating.